src/content/account/billing.md

```diff
@@ -79,26 +79,91 @@ For example, if you want to get a notification when you use 90% of the 35,000 sn
-
-You can generate detailed usage reports for your own bookkeeping. Monthly and yearly reports are available for download as a CSV file. Contact us via our in-app chat or [email](mailto:support@chromatic.com) if you'd like a custom date range or have the report formatted as JSON.
-
-The downloaded CSV includes the following columns:
-
-| Column                      | Description                                                                                                       |
-| --------------------------- | ----------------------------------------------------------------------------------------------------------------- |
-| Date                        | Creation date and time of the build (ISO 8601)                                                                    |
-| App ID                      | Unique Chromatic project identifier                                                                               |
-| Build ID                    | Unique Chromatic build identifier                                                                                 |
-| Repository slug             | Owner and name of the Git repository linked to the project (`<ownerName>:<repoName>`)                             |
-| Branch name                 | Git branch name for which the build was created, prefixed with `<ownerName>:` if the build originates from a fork |
-| Build number                | The incremental number for this build                                                                             |
-| Skipped snapshots           | Number of snapshots that you saved (not got billed for) thanks to [TurboSnap](/docs/turbosnap)                    |
-| Chrome snapshots            | Number of billed snapshots captured in Chrome                                                                     |
-| Firefox snapshots           | Number of billed snapshots captured in Firefox                                                                    |
-| Safari snapshots            | Number of billed snapshots captured in Safari                                                                     |
-| Edge snapshots              | Number of billed snapshots captured in Edge                                                                       |
-| Internet Explorer snapshots | Phasing out in 2023. Number of billed snapshots captured in Internet Explorer.                                    |
+## Usage reports
+
+<div class="callout">
+
+This a new feature that is rolling out in phases. If you don't see it yet, don't worry—it's coming soon. Meanwhile, you can access the [monthly and yearly CSV exports](#export-monthly-and-yearly-usage-data-as-csv).
+
+</div>
+
+Access usage reports for each billing period on the billing page. These reports offer a detailed breakdown of usage and display the invoice statement.
+
+The **usage section** provides a per-project summary of Snapshots and TurboSnaps utilized (only available for billing periods after July 25, 2024).
```

Comment:
Just a small nit here, and feel free to disregard it, but I was wondering if we could use a consistent date format here to align with the rest of the documentation.